net/src/ipc/n3h.rs

```diff
@@ -0,0 +1,294 @@
+//! check that n3h is in the path, or download it
+
+use crate::{connection::NetResult, tweetlog::TWEETLOG};
+use sha2::Digest;
+
+macro_rules! tlog_d {
+    ($($arg:tt)+) => {
+        log_dd!("get_verify_n3h", $($arg)+);
+    }
+}
+
+macro_rules! tlog_e {
+    ($($arg:tt)+) => {
+        log_ee!("get_verify_n3h", $($arg)+);
+    }
+}
+
+#[cfg(not(target_os = "windows"))]
+use std::os::unix::fs::OpenOptionsExt;
```

Comment:
This doesn't seem to be used.

Reply:
one of those weird rust things... this makes the

```rust
open_opts.mode(0o755);
```

below be available on non-windows systems.

Reply:
👍